### ISSUE TITLE
Alphanumeric events not working without prefixing KEY_

### ISSUE THREAD

**Issue by saikat0511** [sezanzeb/input-remapper#258]
Using the latest build from source. My existing presets are not working anymore. `systemctl status input-remapper` shows errors like 
```
ERROR: Don't know what "R" is
```
Changing the keymap to KEY_R fixes it. Same for other alphanumeric keys.

Is this an intended change? [examples.md](https://github.com/sezanzeb/input-remapper/blob/main/readme/examples.md) still shows doing it the old way

**Comment by sezanzeb** [comment]
please post the affected preset from `~/.config/input-remapper/presets/`

**Comment by jonasBoss** [comment]
it is definitely not an intended change. So far I was unable to reproduce this. 
The preset and xmodmap would help to find the issue.

**Comment by saikat0511** [comment]
@sezanzeb The GUI does show the same error

![Screenshot from 2022-01-16 09-42-29](https://user-images.githubusercontent.com/57610828/149647097-6c91b85a-afd2-4132-ac17-8b7d844e0a43.png)

The present in question:
```
{
    "mapping": {
        "1,276,1": [
            "h(k(BTN_LEFT).w(100))",
            "mouse"
        ],
        "2,6,-1": [
            "3",
            "keyboard"
        ],
        "2,6,1": [
            "4",
            "keyboard"
        ],
        "1,275,1": [
            "R",
            "keyboard"
        ]
    }
}
```

xmodmap.json only contains 
```
{}
```

Maybe because I don't have xmodmap installed? I didnt have it installed earlier either. The logs contain a line
```
Optional `xmodmap` command not found. This is not critical.
```

**Comment by sezanzeb** [comment]
That's weird, without xmodmap there is no way for input-remapper to know keys beyond the linux constants https://github.com/torvalds/linux/blob/master/include/uapi/linux/input-event-codes.h

So it's not related to previous changes after all

**Comment by saikat0511** [comment]
I have never installed xmodmap on my machine and input-remapper has worked just fine.

Now I tried installing `xorg-xmodmap` and rebooting but logs still show ```Optional `xmodmap` command not found. This is not critical.``` The command `xmodmap` works fine. The original issue is still there as well

**Comment by sezanzeb** [comment]
what is `xmodmap -pke` printing? Is it reflecting your actual keyboard layout?

**Comment by saikat0511** [comment]
Its printing many lines of keycodes so I guess that's my keyboard layout...

**Comment by sezanzeb** [comment]
I suppose you are running wayland. Is it showing all the keys you would expect on your keyboard?

**Comment by sezanzeb** [comment]
I'll push some improvement to logs to main later, I'll let you know when its there. This might help

**Comment by saikat0511** [comment]
The output of `xmodmap -pke` and input-remapper logs are the same regardless of xorg or wayland

**Comment by sezanzeb** [comment]
Maybe key-mapper could ship with a few default key-name files for various keyboard layouts. Using the output from `locale` it could pick the correct one.

**Comment by sezanzeb** [comment]
Try running

```
rm ~/.config/input-remapper/xmodmap.json
input-remapper-control --symbol-names
cat ~/.config/input-remapper/xmodmap.json
```

to see if this generates the xmodmap.json file

**Comment by sezanzeb** [comment]
usually running input-remapper would replace the xmodmap.json file with new information.

is that file still empty?

**Comment by saikat0511** [comment]
Now the file is populated and my presets work without KEY_ prefix, but systemctl service startup logs are still showing 
```
Optional `xmodmap` command not found. This is not critical.
```
And how come it was working before xmodmap installed 🤔

**Comment by sezanzeb** [comment]
Have you copied the `.config/input-remapper` directory over from a different installation?

I think the service prints this log because it has no real access to the users environment, and I'll hide it in the future on the service. input-remapper-gtk and input-remapper-control are supposed to take care of `xmodmap -pke`.

**Comment by saikat0511** [comment]
Yeah I copied it over from my previous arch install. But like I mentioned before I have never explicitely installed xmodmap and it was working fine with the empty xmodmap.json file until yesterday

If its needed for input-mapper functionality it should be added in the AUR package dependency list since minimal arch installs dont have it.
